src/client/Client.js

```diff
@@ -310,6 +312,36 @@ class Client extends BaseClient {
     });
   }
 
+  /**
+   * Obtains a sticker from Discord.
+   * @param {Snowflake} id ID of the sticker
+   * @returns {Promise<Sticker>}
+   * @example
+   * client.fetchSticker('id')
+   *   .then(sticker => console.log(`Obtained sticker with name: ${sticker.name}`))
+   *   .catch(console.error);
+   */
+  fetchSticker(id) {
+    return this.api
+      .stickers(id)
+      .get()
+      .then(data => new Sticker(this, data));
+  }
```

Comment:
Please use async/await.

Reply:
none of the client methods around this use async/await though? was just being consistent with those

and if these did use async/await it would just be literally adding `async ` and `await ` right?

Reply:
It's not so much about consistency, but slowly converting to async/await. So newer methods should use async/await over thenables.

Reply:
updated, are they good now?

Reply:
We usually prefer splitting those lines for the sake of readability, like in this case it'd be:

```javascript
const data = await this.api.stickers(id).get();
return new Sticker(this, data);
```

Inlining the call to save one variable is something I think we shouldn't do, specially when accessing to properties of it, as it adds a lot of parens.